images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -81,6 +81,7 @@ if (Test-Path $ndkRoot) {
     setx ANDROID_SDK_ROOT $sdkRoot /M
     setx ANDROID_NDK_HOME $ndkRoot /M
     setx ANDROID_NDK_PATH $ndkRoot /M
+    (Get-Content "${ndkRoot}\ndk-build.cmd").replace('%~dp0\build\ndk-build.cmd','"%~dp0\build\ndk-build.cmd"')|Set-Content "${ndkRoot}\ndk-build.cmd"
```

Comment:
I suggest to validate encoding before and after this command.
Packer uses native PowerShell 5.x that works pretty strange with encodings. worth to double check